setup.py

```diff
@@ -22,12 +22,19 @@ def read(fname):
     packages=["pystove"],
     long_description=read("README.md"),
     long_description_content_type="text/markdown",
+    python_requires=">=3.9",
     install_requires=[
         "aiohttp",
     ],
     classifiers=[
         "Development Status :: 3 - Alpha",
         "Topic :: Software Development :: Libraries",
         "License :: OSI Approved :: GNU General Public License v3 or later " "(GPLv3+)",
+        "Programming Language :: Python :: 3",
+        "Programming Language :: Python :: 3.9",
+        "Programming Language :: Python :: 3.10",
+        "Programming Language :: Python :: 3.11",
+        "Programming Language :: Python :: 3.12",
+        "Programming Language :: Python :: 3.13",
```

Comment:
Let's do one thing at a time and keep the PR as small as possible. This is not needed to set up the CI workflow. Also, we may want to get rid of setup.py in favor of pyproject.toml soon anyway, then we can review all this.

Reply:
If we do auto-upgrade to 3.9, we have to state that the library requires 3.9. Isn't it?